python/pyarrow/__init__.py

```diff
@@ -180,7 +180,8 @@ def print_entry(label, value):
                          ListViewType, LargeListViewType,
                          MapType, UnionType, SparseUnionType, DenseUnionType,
                          TimestampType, Time32Type, Time64Type, DurationType,
-                         FixedSizeBinaryType, Decimal128Type, Decimal256Type,
+                         FixedSizeBinaryType,
+                         Decimal32Type, Decimal64Type, Decimal128Type, Decimal256Type,
```

Comment:
You'll need to add `Decimal32Array, Decimal64Array` to line 220 and `Decimal32Scalar, Decimal64Scalar` to line 229